array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,22 @@
+/* exported initial */
+/*
+create new function with parameter array
+create new variable newArray and assign an empty array
+if array is empty, return empty array
+create for loop to iterate through all values of the array except the last
+(i < array.length - 1)
+increment i by 1 each iteration
+for each value push it to newArray
+return newArray
+*/
+
+function initial(array) {
+  var newArray = [];
+  if (array === []) {
+    return [];
+  }
```

Comment:
You can omit this. If the array is empty, the loop would never start, so you would be left returning the empty array at the end of the function, which is the intended result 🤓